guidance/models/_openai.py

```diff
@@ -174,7 +174,7 @@ def _generator(self, prompt):
                 max_tokens=self.max_streaming_tokens,
                 n=1,
                 top_p=self.top_p,
-                temperature=self.temperature,
```

Comment:
Does this mean that `self.temperature` should be deleted from the constructor? Should `top_p` get a similar treatment?

Reply:
self.temperature is still used as the default, but only when the grammar (like a gen function) does not override it. I expect we should give top_p the same control eventually, but that will require updating the grammar to track where is the grammar we get different top_p values

Reply:
This doesn't look like it's going to fall back on `self.temperature` though if `temperature` itself is not provided as an argument to `_generator()`. I think I'm missing something.